service/src/main/java/com/google/auto/service/AutoService.java

```diff
@@ -40,6 +40,6 @@
 @Retention(SOURCE)
 @Target(TYPE)
 public @interface AutoService {
-  /** Returns the interface implemented by this service provider. */
-  Class<?> value();
+  /** Returns the interfaces implemented by this service provider. */
```

Comment:
Maybe `interface(s)`

Reply:
I think `interfaces` is better. I don't know how to pronounce `interface(s)`.

Reply:
Are you saying that the suffix (s) is not common English?

Reply:
I defer to whoever has the more assertive thesaurus

Reply:
I am saying that it is not good English. In the context, there's absolutely no reason any reader would think that the plural implies there must be more than one interface, and there is therefore absolutely no reason to pull readers up short with the unpronounceable `interface(s)`.

You might guess that this is a hobby horse of mine. :-)

Reply:
The reason(s) above make sense to me